hathor/cli/merged_mining.py

```diff
@@ -28,22 +31,34 @@ def execute(args: Namespace) -> None:
     from hathor.merged_mining.status_api import make_app as make_status_app
     from hathor.merged_mining.debug_api import make_app as make_debug_app
     from hathor.merged_mining.bitcoin_rpc import BitcoinRPC
+    from hathor.merged_mining.digibyte_rpc import DigibyteRPC
 
     loop = asyncio.get_event_loop()
 
-    bitcoin_rpc = BitcoinRPC(args.bitcoin_rpc)
+    kwargs: Dict[str, Any] = {}
+    if args.bitcoin_rpc:
+        kwargs['bitcoin_rpc'] = bitcoin_rpc = BitcoinRPC(args.bitcoin_rpc)
+        kwargs['payback_address_bitcoin'] = args.bitcoin_address
+    elif args.digibyte_rpc:
+        kwargs['digibyte_rpc'] = digibyte_rpc = DigibyteRPC(args.digibyte_rpc)
+        kwargs['payback_address_digibyte'] = args.digibyte_address
+    else:
+        raise ValueError('should have at least one parent chain')
```

Comment:
I think we should protect from using more than one parent chain.

Reply:
It took me sometime to understand that we support both at the same time. Maybe add a comment about it somewhere.

Reply:
From what I understood we only support one at a time. That's why we get only one of the arguments from the parser: `parent_chain = parser.add_mutually_exclusive_group()`

Reply:
Yes, it's one at a time at the command line level. Internally it almost works with both of them, but there are still some unresolved questions when using multiple parent chains, so I opted to block it at the command line level.